src/ADAL.PCL.Desktop/WebUI.cs

```diff
@@ -47,9 +47,17 @@ internal abstract class WebUI : IWebUI
             var sendAuthorizeRequest = new Action(
                 delegate
                 {
+                    string requestId = Telemetry.GetInstance().CreateRequestId();
+                    Telemetry.GetInstance().StartEvent(requestId, "ui_event");
+
+                    UIEvent UiEvent = new UIEvent();
+
                     authorizationResult = this.Authenticate(authorizationUri, redirectUri);
+
+                    Telemetry.GetInstance().StopEvent(requestId, UiEvent, "ui_event");
                 });
```

Comment:
We had a conversation about this before. This is not where the ui event finishes. This is just a delegate declaration. The actual execution of the call happens on line 69 and 89.